lms/templates/dashboard/index.html.jinja2

```diff
@@ -1,6 +1,10 @@
 {% extends "lms:templates/base.html.jinja2" %}
 
 
+{% block title %}
+  {{ assignment.title }} - Hypothesis
```

Comment:
We discussed showing only a few words of the assignment title, but browsers have their own logic to just show whatever fits in the window or tab title, so I think we can set the whole title here and avoid extra logic.